### PR TITLE
Add NetEase Cloud Music

### LINKED ISSUE #3800: netease-cloudmusic please!
**Name: netease-cloudmusic**
**Website: https://music.163.com/**
**Alexa rank: 139**
  

**Official resources for icon and color:**
  
  base64:

data:image/svg+xml;base64,PHN2ZyBjbGFzcz0iaWNvbiIgdmlld0JveD0iMCAwIDEwMjQgMTAyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB3aWR0aD0iOTYiIGhlaWdodD0iOTYiPjxkZWZzPjxzdHlsZS8+PC9kZWZzPjxwYXRoIGQ9Ik02MjcuMDg2IDUuMTE1YzI4LjEzMi03LjY3MiA1OC44MjItNy42NzIgODYuOTUzIDAgMzMuMjQ3IDcuNjcyIDYzLjkzNyAyMy4wMTcgODkuNTEyIDQzLjQ3NyAxMC4yMyA3LjY3MyAxNy45MDIgMTUuMzQ0IDIzLjAxNyAyOC4xMzEgNy42NzIgMTcuOTAzIDUuMTE0IDM4LjM2My01LjExNSA1My43MDgtNy42NzIgMTIuNzg3LTIzLjAxNyAyMy4wMTctNDAuOTIgMjUuNTc0LTEyLjc4NyAyLjU1OC0yNS41NzQgMC0zOC4zNjItNy42NzItNS4xMTUtMi41NTgtMTAuMjMtMTAuMjMtMTcuOTAyLTEyLjc4Ny0xNy45MDItMTAuMjMtMzUuODA0LTIwLjQ2LTU2LjI2NC0xNy45MDMtMTUuMzQ1IDAtMjguMTMyIDcuNjczLTM1LjgwNCAxNy45MDMtMTAuMjMgMTAuMjMtMTIuNzg4IDIzLjAxNy0xMC4yMyAzNS44MDQgNy42NzIgMjUuNTc0IDEyLjc4NyA1My43MDYgMjAuNDYgNzkuMjgxIDUxLjE1IDIuNTU4IDk5Ljc0IDE1LjM0NSAxNDMuMjE4IDQwLjkyIDQwLjkyIDI1LjU3NSA3OS4yOCA1OC44MjEgMTA5Ljk3IDk3LjE4MyAyNS41NzUgMzMuMjQ3IDQ2LjAzNSA3MS42MSA1Ni4yNjUgMTEyLjUzIDEyLjc4NiA0My40NzYgMTcuOTAxIDg5LjUxIDEyLjc4NiAxMzIuOTg2LTIuNTU3IDM4LjM2My0xMC4yMyA3NC4xNjYtMjMuMDE2IDEwOS45NzEtMzMuMjQ3IDg0LjM5Ni05Mi4wNyAxNjEuMTItMTcxLjM1IDIwOS43MTMtNTYuMjY1IDM1LjgwMy0xMjIuNzYgNTguODIxLTE4OS4yNTMgNjYuNDkzLTQ2LjAzNCA1LjExNS05Mi4wNjkgNS4xMTUtMTM4LjEwMi0yLjU1Ny05NC42MjctMTUuMzQ1LTE4MS41OC02MS4zOC0yNTAuNjMxLTEzMC40MzEtNjYuNDk1LTY2LjQ5My0xMTIuNTMtMTUzLjQ0OC0xMzIuOTktMjQ1LjUxNi03LjY3MS02OS4wNTItNy42NzEtMTM4LjEwMyA3LjY3My0yMDcuMTU0IDE3LjkwMy04MS44NCA2MS4zOC0xNjEuMTIgMTE3LjY0NC0yMjIuNSA0OC41OTItNTEuMTUgMTA3LjQxNC04OS41MTEgMTcxLjM1LTExNy42NDMgNy42NzItMi41NTggMTIuNzg3LTUuMTE1IDIwLjQ2LTcuNjczIDE1LjM0NC0yLjU1NyAzMC42OSAwIDQzLjQ3NyAxMC4yMyAxNy45MDIgMTIuNzg4IDI1LjU3NCAzMy4yNDggMjMuMDE3IDUzLjcwNy0yLjU1NyAyMC40Ni0xNy45MDIgMzguMzYzLTM1LjgwNSA0Ni4wMzQtNjMuOTM3IDI1LjU3NS0xMjIuNzU4IDY5LjA1Mi0xNjMuNjc4IDEyMi43Ni0zOC4zNjIgNTMuNzA1LTYzLjkzNiAxMTIuNTI3LTcxLjYwOCAxNzMuOTA2LTcuNjcyIDYxLjM4IDAgMTIyLjc1OCAyMC40NiAxODEuNTggMzAuNjkgODQuMzk2IDk0LjYyNiAxNTYuMDA0IDE3My45MDcgMTk2LjkyNCA0OC41OTIgMjUuNTc1IDEwMi4yOTggMzguMzYyIDE1Ni4wMDUgMzguMzYyIDQzLjQ3NyAwIDg5LjUxMS03LjY3MiAxMzAuNDMtMjMuMDE3IDM1LjgwNS0xMi43ODcgNzEuNjEtMzMuMjQ3IDk5Ljc0MS01OC44MjIgMjguMTMzLTIzLjAxNiA1MS4xNS01My43MDYgNjYuNDk1LTg0LjM5NiA3LjY3Mi0xNS4zNDUgMTcuOTAxLTMzLjI0NyAyMC40Ni01MS4xNSAxNS4zNDQtNTEuMTQ5IDE3LjkwMS0xMDcuNDEzIDIuNTU2LTE1OC41NjEtMTIuNzg2LTQzLjQ3OC0zOC4zNjEtODEuODQtNzEuNjA5LTEwOS45NzEtMTUuMzQ0LTEyLjc4Ny0zMC42OS0yNS41NzUtNDguNTkyLTM1LjgwNS0xNS4zNDQtNy42NzItMzAuNjktMTUuMzQ1LTQ4LjU5MS0xNy45MDIgMTIuNzg4IDQ2LjAzNCAyMy4wMTggOTIuMDcgMzUuODA0IDEzNS41NDUgMi41NTggMTAuMjMgNS4xMTUgMjMuMDE4IDUuMTE1IDMzLjI0OCAyLjU1OCA0Ni4wMzMtMTUuMzQ0IDk0LjYyNS00Ni4wMzQgMTMwLjQzLTI4LjEzMiAzMy4yNDYtNjkuMDUyIDU4LjgyMS0xMTIuNTI4IDY2LjQ5NC00Ni4wMzQgMTAuMjMtOTcuMTg0IDAtMTM4LjEwMy0yNS41NzUtMzguMzYyLTI1LjU3NC02Ni40OTQtNjMuOTM2LTgxLjg0LTEwNC44NTYtNy42NzItMjMuMDE3LTEyLjc4Ny00OC41OTEtMTIuNzg3LTc0LjE2Ni0yLjU1Ni01Ni4yNjQgMTIuNzg4LTEwOS45NzEgNDMuNDc4LTE1Ni4wMDUgMzUuODA0LTUzLjcwNyA5NC42MjUtOTIuMDcgMTU4LjU2Mi0xMDkuOTcxLTUuMTE1LTE3LjkwMi0xMC4yMy0zNS44MDUtMTIuNzg3LTUzLjcwNy0xMi43ODctMzguMzYxLTEwLjIzLTgxLjgzOSA3LjY3Mi0xMTUuMDg2IDEwLjIzLTIwLjQ2IDIzLjAxOC0zOC4zNjEgNDAuOTItNTEuMTUgMjMuMDE2LTIwLjQ2IDQzLjQ3Ni0zMy4yNDYgNjYuNDk0LTQwLjkxOE00NzguNzUzIDQxOS40MjRjLTE3LjkwMyAxNy45MDItMjguMTMzIDQwLjkyLTMzLjI0NyA2My45MzYtNS4xMTQgMjAuNDYtNS4xMTQgNDMuNDc3IDAgNjYuNDk1IDUuMTE0IDIzLjAxNiAxNy45MDIgNDYuMDMzIDM4LjM2MiA2MS4zOCAxNS4zNDUgMTAuMjI4IDM1LjgwNCAxNS4zNDMgNTYuMjY0IDEwLjIyOCAzNS44MDQtNS4xMTUgNjMuOTM2LTM4LjM2MiA2My45MzYtNzQuMTY2LTIuNTU3LTcuNjcyLTIuNTU3LTE3LjkwMi01LjExNS0yNS41NzUtMTIuNzg3LTQ4LjU5Mi0yNS41NzMtOTkuNzQxLTM4LjM2MS0xNDguMzMzLTMwLjY5IDcuNjczLTU4LjgyMiAyMy4wMTgtODEuODQgNDYuMDM1eiIgZmlsbD0iI2ZmZiIvPjwvc3ZnPg==

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
NetEase Cloud Music

**Issue:** closes #3800

**Similarweb rank:** [3840](https://www.similarweb.com/website/music.163.com)

### Checklist
- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
Taking a stab at clearing out one of our longest standing open requests. Icon & colour from [PNG sprite](https://s2.music.126.net/style/web2/img/frame/topbar.png?16a3cd7b49d34cad17bf2a42966a67f6) found at [source](https://music.163.com); Simplified Chinese translation from first `h1` tag in HTML source.

### CODE REVIEW

**Review by adamrusted — APPROVED** [review]
Another nigh-on-perfect conversion, thanks @PeterShaggyNoble!
image

Will await approval from another maintainer before we merge though.

**Comment by LitoMore** [comment]
I just extracted the SVG from their mobile site https://y.music.163.com

CleanShot 2023-12-14 at 19 52 29@2x

Here is the path:

```
M40.4991735,31.4579385 C40.219059,30.4818472 39.9378671,29.500369 39.682532,28.5555211 C39.0059476,28.6869595 38.3336728,28.8798076 37.6732489,29.1340654 C35.2147053,30.0810679 33.2732961,32.1474512 32.6074855,34.5305794 C32.3844712,35.3278284 32.3079784,36.1584758 32.3812391,36.997742 C32.5266832,38.6558045 33.3950382,40.1888928 34.7051123,41.1003424 C35.5066708,41.656262 36.8188996,42.2186458 38.5200567,41.7521473 C39.4465893,41.4946575 40.2643083,40.9548983 40.8169958,40.2330647 C42.0548865,38.615942 41.8986688,36.6346704 41.5506804,35.2567225 C41.2522507,34.0780867 40.8773281,32.7744768 40.4991735,31.4579385 M55.0974497,49.521016 C51.4548836,55.39157 44.424009,59.1849671 37.1852032,59.1849671 C25.07025,59.1849671 15.2134508,49.3292453 15.2134508,37.214292 C15.2134508,36.1293869 15.2974852,35.0337082 15.4623218,33.957422 C15.5959149,33.0922991 15.7822988,32.2282535 16.017164,31.391142 C16.2477198,30.5648042 16.5321438,29.7427758 16.8618171,28.9487588 C17.1871808,28.1622834 17.5631807,27.387659 17.9790431,26.6464329 C18.3884412,25.9127483 18.8484755,25.1930695 19.3451401,24.5067889 C19.8364178,23.8269725 20.3740223,23.167626 20.9450249,22.5470646 C21.5106408,21.9286579 22.1215059,21.3350305 22.7571504,20.7855751 C23.3917176,20.2361197 24.0661472,19.7168305 24.7632014,19.2427905 C25.4634877,18.7655185 26.1971723,18.3259541 26.94594,17.9348712 C27.3564155,17.7215532 27.7744325,17.5190088 28.1989138,17.3315476 C28.4359338,17.2281207 28.6675669,17.1290032 28.9013548,17.03635 C30.1392456,16.5407627 31.5419729,17.1451637 32.0354054,18.3830544 C32.5277606,19.6198678 31.9255143,21.0215178 30.6876236,21.5149503 C30.5077039,21.5871336 30.3288615,21.6636264 30.1521739,21.7411966 C29.8225007,21.8866407 29.4982142,22.0428584 29.1782373,22.2109271 C28.5943062,22.5147436 28.0222262,22.8573453 27.4781576,23.2279583 C26.9351663,23.5985714 26.4083355,24.0036601 25.9127483,24.4324508 C25.4171611,24.8612415 24.9420437,25.3234304 24.49817,25.805012 C24.0542962,26.2909029 23.6352017,26.8048053 23.2516603,27.3348682 C22.8648868,27.8692405 22.5061247,28.4305469 22.1861478,29.002627 C21.8629387,29.5790165 21.5709732,30.1823401 21.3167154,30.7942826 C21.0613802,31.4116119 20.8405207,32.0494111 20.6595236,32.6925972 C20.4774492,33.3444021 20.3320051,34.0155996 20.2296555,34.6889519 C20.1003719,35.5239086 20.0357301,36.3728711 20.0357301,37.214292 C20.0357301,46.670312 27.7291833,54.3637652 37.1852032,54.3637652 C42.7939579,54.3637652 48.2163287,51.4645798 51.0002361,46.9773606 C54.1698397,41.8706573 53.3488887,35.0218572 49.0922253,31.0453083 C47.8661855,29.9000708 46.3578766,29.1038992 44.7224387,28.6837274 C44.8592639,29.1631542 44.9971664,29.645813 45.1350689,30.1273946 C45.517533,31.4590159 45.911848,32.8358865 46.2253608,34.0737773 C47.0872517,37.4782461 46.5270226,40.7071047 44.6470232,43.1634936 C43.4586912,44.7170519 41.739219,45.8655214 39.804274,46.3998937 C37.0699253,47.1529709 34.2838632,46.6789309 31.9535258,45.0607308 C29.4809764,43.3412586 27.8444612,40.4851678 27.577275,37.415759 C27.4533782,35.9957939 27.5826618,34.5876797 27.9640485,33.2302016 C29.0317158,29.4109478 32.0892736,26.11637 35.9408483,24.6339178 C36.7628767,24.3171729 37.6010656,24.0683019 38.4446412,23.8873048 C38.343369,23.5048408 38.1494436,22.7690014 38.0373978,22.349907 C36.9083208,18.1794327 38.7990938,14.505623 42.7411671,13.2030904 C43.7420378,12.8734172 44.8118599,12.7624487 45.8590572,12.8831134 C46.3352519,12.938059 46.8049824,13.0393311 47.2607072,13.1858526 C48.2303344,13.4961333 49.1428614,13.99603 49.9164084,14.6596859 C50.318265,15.0044423 50.7082706,15.3998347 50.9776115,15.8534048 C50.9883851,15.8727973 50.9991588,15.8911125 51.0099324,15.910505 C51.0260929,15.9385165 51.0411759,15.9676053 51.056259,15.9956168 C51.2243278,16.32529 51.3202131,16.6980578 51.3202131,17.0934502 C51.3202131,18.4250716 50.2406948,19.5045899 48.9090735,19.5045899 C48.273429,19.5045899 47.6959621,19.258951 47.2650167,18.856017 C46.9989079,18.6093007 46.7715842,18.3248768 46.4796186,18.1040172 C46.2016588,17.8950087 45.8838366,17.7355589 45.5466218,17.6461377 C45.2244901,17.561026 44.8861979,17.5437882 44.5576021,17.5998111 C43.8325364,17.7204758 43.1785767,18.1675817 42.8295109,18.8172319 C42.749786,18.9659081 42.6765253,19.1135069 42.6291213,19.2751114 C42.4599752,19.8568877 42.5332359,20.5097701 42.6916084,21.0850822 C42.9006169,21.8629387 43.1721125,22.9015172 43.3509549,23.5802562 C46.7252575,23.8765312 49.9250273,25.224313 52.3835709,27.5212521 C58.3877181,33.1289294 59.5297234,42.3813277 55.0974497,49.521016 M36.5,0 C16.3425278,0 0,16.3414505 0,36.5 C0,56.6585495 16.3425278,73 36.5,73 C56.6585495,73 73,56.6585495 73,36.5 C73,16.3414505 56.6585495,0 36.5,0
```

**Comment by adamrusted** [comment]
Damn you Apple users and your unified ecosystem! _\*shakes fist angrily\*_

**Comment by LitoMore** [comment]
@adamrusted You can use Chrome Android and Chrome Desktop to inspect it as well 😄

**Comment by adamrusted** [comment]
My angry fist-shaking at Apple users still stands! 😅

File: _data/simple-icons.json
Comment (LitoMore) [inline _data/simple-icons.json:9320]: The name should be `NetEase Music`. Here is the app name on my phone:

image

**Review by LitoMore — CHANGES_REQUESTED** [review]

**Comment by LitoMore** [comment]
@PeterShaggyNoble Would you mind me helping you update this PR with some new commits?

**Comment by PeterShaggyNoble** [comment]
You're some jerk, @LitoMore! 😛 😆 I bet you've been sitting on that all these years just _waiting_ for someone to try to recreate it from raster! 😆

Updated everything based on that new source. On the name, it looks like NetEase Music is the name name of the _app_, but NetEase Cloud Music is actually the full name, going by their Wikipedia entry and the heading on their main website. But, as you're more familiar with the brand, I'll defer to you.

**Comment by LitoMore** [comment]
I found this: https://music.163.com/#/topic?id=18575148.

In step 2, they call themselves "NetNease Cloud Music".

**Review by LitoMore — APPROVED** [review]

**Comment by LitoMore** [comment]
They have some logos with rounded square backgrounds. For example the one at the bottom of https://y.music.163.com/m/.

image

How about dropping the circle background and using the inside shape only?

**Comment by PeterShaggyNoble** [comment]
> How about dropping the circle background and using the inside shape only?

I haven't seen it in use anywhere without a background shape, so we'll need to include one and, as best as I can tell, the one with the circle is for the overall brand/service, while the one with the "squircle" is just for the app.

**Pull request merged**
